### ISSUE TITLE
Cannot change dark version colors via ColorScheme autocmd

### ISSUE THREAD

**Issue by kristijanhusak** [projekt0n/github-nvim-theme#12]
Hi,

I'm not able to change the colorscheme colors via `ColorScheme` autocmd when themeStyle is `dark`. For some strange reason `light` version works. I have this:

```lua
vim.cmd[[augroup vimrc_colorscheme]]
  vim.cmd[[autocmd ColorScheme * lua kris.colorscheme.setup_github()]]
vim.cmd[[augroup END]]

function _G.kris.colorscheme.setup_github()
  local colors = require('github-theme.colors').setup()
  vim.cmd(string.format('hi StatusLine guibg=%s', colors.white))
end

require("github-theme").setup({
  themeStyle = 'light',
})
```

But when I change `themeStyle` to `dark`, autocmd is never triggered:

```lua
require("github-theme").setup({
  themeStyle = 'dark',
})
```

Related issue for statusline: #11

**Comment by kristijanhusak** [comment]
From what I figured, it has to do something with the default `background` setting.
For example, when I use dark kitty colors, default `background` is `dark`.
Then autocmd is not triggered for both `light` and `dark` themeStyle.

When I use light kitty colors, default background is `light`, and both `dark` and `light` themeStyles properly trigger autocmd.

It worked for me in the initial comment only for light because i toggle both kitty and neovim colors at the same time. When I tried dark colorscheme with light kitty it also worked.
